roleGroup.py

```diff
@@ -81,13 +81,13 @@ def fromJson(self, data: dict, emojis: Tuple[Emoji], manual_load=False):
                 # Only create new roles if we're not loading data manually from
                 # the command channel
                 role = Role(roleData["name"], roleEmoji,
-                            roleData["displayName"])
+                            roleData["show_name"])
```

Comment:
I have fixed it for loading from the event.json
`#KOSI START this only needs to be loaded once
    def getCorrectedName(self, roleData):
        if "displayName" in roleData:
            show_name =  roleData["displayName"]
        else:
            how_name = roleData["show_name"]
        return show_name      
#KOSI END`

I dont know when and where you load from the archives.json. Also I dont know if I need to change anything else there.

Reply:
apparently using code in the comments is very bad

Reply:
#KOSI START this only needs to be loaded once
    def getCorrectedName(self, roleData):
        if "displayName" in roleData:
            show_name =  roleData["displayName"]
        else:
            how_name = roleData["show_name"]
        return show_name      
#KOSI END

Reply:
So your saying just convert the JSON data to the new show_name tag?

Reply:
yes
We could do that by hand as well. But I dont know how big the file is

Reply:
Its very big, 70,000 lines 1.8MB~ but it should be fine

Reply:
This will do the job for now:
https://github.com/zeusops/operationbot/blob/d2ff26843e12106a6e2c7f14582189a642d790e4/roleGroup.py#L100-L107

Ideally I'd like to check the DB version and do a conversion based on that but I don't want to have this PR to wait for that implementation